### DISCUSSION TITLE
Issues with Umlauts (Special Chars) from Output

### DISCUSSION THREAD

**Discussion by SenorCerveza** [rodneyviana/ODSyncUtil#7] [Q&A] [answered]
We have issues that the binary drops an Exception as soon the Output drops any umlauts like ä ö or something.

Can this be resolved somehow? We tried to start the binary in any way UTF8 based like chcp 65001 but it doesn't work.

Kindly

**Comment by SenorCerveza** [comment]
We also just tried to change the System wide locale to UTF8 (Windows Beta feature on Windows 11 / Server 2022) and that didn't helped. Event when redirecting the Output from the binary to a File in a separate CMD with changed codepage the file abrubt stops as soon an umlaut appears in the output. @rodneyviana

**Comment by rodneyviana** [comment, marked as answer]
Just checking if you are still experiencing it

**Reply by SenorCerveza** [reply]
Sorry for the delay. Yes its working flawless now :)

Thanks again
